code/modules/admin/admin_verbs.dm

```diff
@@ -747,7 +747,7 @@ var/list/admin_verbs_cciaa = list(
 	set name = "Toggle Build Mode Self"
 	set category = "Special Verbs"
 
-	if(!check_rights(R_ADMIN))
+	if(!check_rights(R_ADMIN|R_DEV))
```

Comment:
```suggestion
	if(!check_rights(R_ADMIN|R_SPAWN))
```
Spawning is restricted to R_SPAWN

Reply:
This isn't spawning, this is the build mode, what was R_DEV before the update too?

Reply:
AFAIK R_DEV has been removed quite a while from buildmode and moved to R_SPAWN given that the primary purpose of buildmode is to efficiently spawn stuff.